pkg/apis/admissionregistration/fuzzer/fuzzer.go

```diff
@@ -32,6 +32,9 @@ var Funcs = func(codecs runtimeserializer.CodecFactory) []interface{} {
 			obj.FailurePolicy = &p
 			s := admissionregistration.SideEffectClassUnknown
 			obj.SideEffects = &s
```

Comment:
fuzzing isn't typically concerned with validation rules, just with fields that are assigned defaults if nil, so:

```
if obj.TimeoutSeconds == nil {
  i := int32(30)
  obj.TimeoutSeconds = &i
}
```